docs/explanations/curator_data_model.md

```diff
@@ -0,0 +1,389 @@
+# CSV data model description
+
+The Curator-Extension (formerly Schematic) data model is used to create JSON Schemas for Curator. See [JSON Schema documentation](https://json-schema.org/). This is used for the DCCs that prefer working in a tabular format (CSV) over JSON or LinkML. A data model is created in the format specified below. Then the Curator-Extension in the Synapse Python Client can be used to convert to JSON Schema.
+
+A link will be provided here to documentation for converting CSV data models to JSON Schema in the near future.
+
+## Understanding the Structure
+
+A data model describes real world entities(data types) and attributes that you want to collect data for. For example you might want to describe a Patient, and you want to collect their age, gender and name.
+
+The CSV data model described in this tutorial formalizes this structure:
+
+- The CSV data model describes one or more data types.
+- Each row describes either a data type, or an attribute.
+
+Here is the Patient described above represented as a CSV data model:
+
+| Attribute | DependsOn |
+|---|---|
+| Patient   | "Age, Gender, Name" |
+| Age       |                     |
+| Gender    |                     |
+| Name      |                     |
+
+The end goal is to create a JSON schema that cna be used in Curator. A JSON Schema consists of only one data type and their attributes. Converting the above data model to JSON Schema results in:
+
+```json
+{
+  "description": "TBD",
+  "properties": {
+    "Age": {
+      "description": "TBD",
+      "title": "Age"
+    },
+    "Gender": {
+      "description": "TBD",
+      "title": "Gender"
+    },
+    "Name": {
+      "description": "TBD",
+      "title": "Name"
+    }
+  }
+}
+```
+
+## CSV Data model columns
+
+Note: Individual columns are covered later on this page.
+
+Defining data types:
+
+- Put a unique data type name in the `Attribute` column
+- List its attributes (minimum 1) in the `DependsOn` column (comma-separated)
+- Optionally add a description to the `Description` column.
```

Comment:
Are all other columns optional for data types, not just the Description column?

Reply:
The other columns aren't used for data types AFAIK.